src/extension.ts

```diff
@@ -172,8 +174,8 @@ export function activate(context: vscode.ExtensionContext) {
 					return
 				}
 
-				if (token) {
-					await visibleProvider.handleAuthCallback(token)
+				if (token && apiKey) {
```

Comment:
This will change functionality and we will not longer have a branch for `token` in this condition cascade.  Is this intended?